src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/tts_get_language_response.cc

```diff
@@ -66,6 +78,13 @@ void TTSGetLanguageResponse::Run() {
 
   hmi_capabilities_.set_active_tts_language(language);
 
+  std::vector<std::string> sections_to_update;
```

Comment:
@ydementieiev here brace initialization can be used

Reply:
@AByzhynar I changed it in all needed [place](https://github.com/LuxoftSDL/sdl_core/pull/35/commits/cdab57c9b89acd94155de176f4a223e55f72602a)